libmscore/dynamic.h

```diff
@@ -21,6 +21,59 @@ namespace Ms {
 class Measure;
 class Segment;
 
+//-----------------------------------------------------------------------------
+//   Dyn
+//    see: http://en.wikipedia.org/wiki/File:Dynamic's_Note_Velocity.svg
+//-----------------------------------------------------------------------------
+
+struct Dyn {
+      int velocity;      ///< associated midi velocity (0-127, -1 = none)
+      bool accent;       ///< if true add velocity to current chord velocity
+      const char* tag;   // name of dynamics, eg. "fff"
+      const char* text;  // utf8 text of dynamic
+      int changeInVelocity;
+      };
+
+// variant with ligatures, works for both emmentaler and bravura:
+
+static Dyn dynList[] = {
```

Comment:
This resulte in a ton )>120) of warnings reg. a defined but unused variable

Reply:
Variables should never ever get **defined** in header files...

Reply:
Hmm, making that `static const Dyn dynList[] = {` fixes the compiler warnings

Reply:
Hmm, fair point. I couldn't figure out a simple way to make this list available to the XML importer without declaring it in the header file, but I understand defining it there is a bit dirty. Perhaps a vector rather than array, to define it without having to know the size?

Reply:
Just adding the `const` appeases the compiler

Reply:
I'll add a PR with that then—thanks!

Reply:
#8478

Reply:
Maybe you can add fixes for the other warnings, from your other PRs?